.gitignore

```diff
@@ -1,2 +1,3 @@
 # .gitignore
-node_modules/
\ No newline at end of file
+node_modules/
+test.md
```

Comment:
- [ ] Good job adding `node_modules` and `test.md` to your `.gitignore` file :+1:, but as a [project requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) you should make sure that test.md is actually ignored by git and is not present in your GitHub repository :100: